packages/react-composites/tests/browser/call/app/index.tsx

```diff
@@ -47,7 +47,7 @@ function App(): JSX.Element {
   useEffect(() => {
     const initialize = async (): Promise<void> => {
       if (mockCallState) {
-        setCallAdapter(wrapAdapterForTests(new MockCallAdapter(mockCallState)));
```

Comment:
Needed to remove this because [this wrapper unsets the 'speakingWhileMicrophoneIsMuted' diagnostic](https://github.com/Azure/communication-ui-library/blob/a6c7201daab7a6545520a11de24a1e7611197a5e/packages/react-composites/tests/browser/call/app/index.tsx#L110).

Reply:
Yes, that is necessary in live e2e tests because the isMuted diagnostic is flakily set in our test setup.

It makes sense that it isn't needed for mockAdapter.